src/title.rs

```diff
@@ -1,25 +1,76 @@
+#![warn(missing_docs)]
+//! This module holds the [Title] element and its related configuration types.
```

Comment:
nit: (Across the entire PR - add backticks in the links to code items. e.g. 
```suggestion
//! This module holds the [`Title`] element and its related configuration types.
```